src/AppInstallerCLICore/PortableInstaller.cpp

```diff
@@ -141,6 +141,7 @@ namespace AppInstaller::CLI::Portable
                 AICLI_LOG(Core, Info, << "Portable install executed in user mode. Adding package directory to PATH.");
                 CommitToARPEntry(PortableValueName::InstallDirectoryAddedToPath, InstallDirectoryAddedToPath = true);
             }
+            m_stream << Resource::String::PortableAliasAdded << filePath.stem() << std::endl;

```

Comment:
So if you choose to print it here, the message will only be displayed if the symlink was successfully created. If it fails, `InstallDirectoryAddedToPath` will be set to true and this block of code will never be hit again, thus hiding your command alias message.  Here is what I would do: 

```
else if (entry.Filetype == PortableFileType.Symlink)
{
   if (!InstallDirectoryAddedToPath) { ... }

  m_stream << Resource::String::PortableAliasAdded << filePath.stem() << std::endl;
}
```

That way you can guarantee the message is always displayed to the user for each symlink that was intended to be created.

Reply:
Ahh, I see. I thought placing it outside the if-else was enough, but I can see how it could fail now